modules/appeals_api/config/schemas/v2/200995_headers.json

```diff
@@ -103,5 +128,19 @@
         { "$ref": "#/definitions/nonBlankString" }
       ]
     }
+  },
+  "additionalProperties": false,
```

Comment:
I believe we stopped using the `additionalProperties` key in supplemental claims. @cilestin, can you confirm?

Reply:
Confirmed, we should no longer be adding `additionalProperties` to our schemas.